src/playlist-selectors.js

```diff
@@ -103,17 +120,22 @@ export const comparePlaylistResolution = function(left, right) {
 
   rightWidth = rightWidth || window.Number.MAX_VALUE;
 
+  const leftBandwidth = getBandwidth(left, useAverageBandwidth);
+  const rightBandwidth = getBandwidth(right, useAverageBandwidth);
+
   // NOTE - Fallback to bandwidth sort as appropriate in cases where multiple renditions
-  // have the same media dimensions/ resolution
-  if (leftWidth === rightWidth &&
-      left.attributes.BANDWIDTH &&
-      right.attributes.BANDWIDTH) {
-    return left.attributes.BANDWIDTH - right.attributes.BANDWIDTH;
+  // have the same media dimensions/ resolution.
+  if (leftWidth === rightWidth && leftBandwidth && rightBandwidth) {
```

Comment:
That's a good point. We should be able to remove that comparison. The only reason we might be inclined to keep it is for backwards compatibility since `comparePlaylistResolution` is exposed.